### PR TITLE
Added tests for BT/BTS/BTR/BTC instructions

### PR DESCRIPTION
**Pull request by AlexAltea**
Follow-up patch for #122.

Signed-off-by: Alexandro Sanchez Bach <asanchez@kryptoslogic.com>

### CODE REVIEW

File: tests/test_emulator.cpp [hunk 1]
```diff
@@ -488,6 +488,18 @@ class EmulatorTest : public testing::Test {
         test_insn_mN_rN<N>(insn_name, tests, false);
     }
 
+    template <int N>
+    void test_bt(const char* insn_name,
+        const std::vector<test_alu_2op_t>& tests) {
```
Comment (raphaelning) [inline tests/test_emulator.cpp]:
```suggestion
                 const std::vector<test_alu_2op_t>& tests) {
```

Comment (raphaelning) [reply tests/test_emulator.cpp]: I'm ready to merge this PR, but I don't know how to amend your original commit and apply this coding style suggestion... Let me know if you can do it easily :-)

Comment (AlexAltea) [reply tests/test_emulator.cpp]: Sorry, my editor messed the function argument indentation (once again). I'll modify the settings so it doesn't happen another time. Fixed at f4884d4.

**Comment by AlexAltea** [comment]
All done!

> and unnecessarily introducing the `readonly_dst` parameter. Sorry!

It's alright, I will probably fix it in a future PR since I wanted to improve the flexibility of the *test_insn_xN_xN* functions anyway. Specifically, I wanted to template them over both integers N and M: *test_insn_xN_xM*. But I'll queue that for a future PR.

**Pull request merged**